config/app.yml

```diff
@@ -45,3 +45,11 @@ all:
   # to a different string value to make sure the proper workers take the job for
   # their related instance.
   workers_key:
+
+  # Specify password_hash algorithm constant
+  # See: https://www.php.net/manual/en/password.constants.php
+  password_hash_algorithm: PASSWORD_ARGON2I
+
+  # Specify password_hash algorithm options (as JSON)
+  # See: https://www.php.net/manual/en/function.password-hash.php
+  password_hash_algorithm_options: {"memory_cost": "2048", "time_cost": "4", "threads": "3"}
```

Comment:
Out of curiosity, how did you chose these values for cost? :)

Reply:
I just copied them from some blog post so no apparent reason. I'm open to cranking up the difficulty!

Reply:
Probably fine, I have no idea what reasonable values are. 🤷‍♂ 😄